src/components/navigation/constants.ts

```diff
@@ -1,6 +1,6 @@
 export const navLinks = [
-  { href: "/about", name: "About" },
-  { href: "/shop", name: "Shop" },
-  { href: "#contacts", name: "Contacts" },
-  { href: "/cart", name: "Cart" },
+  { href: "/about", title: { en: "About", uk: "Про мене" } },
```

Comment:
You can also use title as some key, and then load it from locales:
 
```tsx
{ href: "/about", title: "about" }
...
const key = navLinks[index].title
const text = dictionary[key]
<p>{text}</p>
```

Because with current implementation, you lose all benefits of extracting constants to `en/uk.json`